lib/audit/catalog_to_moab.rb

```diff
@@ -40,52 +40,45 @@ def self.check_version_all_dirs_profiled(last_checked_b4_date)
     profiler.print_results_flat('C2M_check_version_all_dirs')
   end
 
-  # TODO:  you need to write tests for this!!!
-  # FIXME:  temporarily turning off rubocop until we migrate the code to its final home
-  # rubocop:disable all
-  private_class_method def self.check_catalog_version(preserved_copy, storage_dir)
+  private_class_method def self.check_catalog_version(preserved_copy, _storage_dir)
```

Comment:
would there be a performance boost if we kept the `storage_dir` as an argument and did not do the active record look up for the storage dir every time we needed it?

Reply:
I would guess it would improve performance, but since we worked hard on seeding code to remove unnec. per record db lookups and the performance difference was minimal ... shrug.

Happy to change the code to use the arg;  happy to change the code to not have the arg.

Reply:
i think this is fine for now as-is in this PR... we probably will want to pass in either the endpoint (from which we can figure out `storage_dir`), or just keep passing in `storage_dir` since we already have it.  it does seem like `check_catalog_version` will need to know something `storage_dir` related since it needs to see if the thing is there on disk.

my gut feeling at the moment is that it'll become more obvious whether to pass in `storage_dir` as we flesh out `check_catalog_version`, and i think i'd rather just get this merged and take the leading `_` back off the var name if it turns out we do want to use this param as-is.  i don't think this change is painting us into a corner or anything.